QSB/TransformSync/PlayerTransformSync.cs

```diff
@@ -29,9 +29,20 @@ protected override Transform InitLocalTransform()
         protected override Transform InitRemoteTransform()
         {
             var body = Instantiate(GetPlayerModel());
-
             GetComponent<AnimationSync>().InitRemote(body);
 
+            var physicsBody = new GameObject("PlayerBodySync");
+
+            var collider = physicsBody.AddComponent<CapsuleCollider>();
+            collider.radius = 1;
+            collider.height = 2;
+            collider.center = Vector3.up * 1;
```

Comment:
I recommend adding a cylinder collider to the top half of the capsule, to make it easy to stand on players heads 😁

Reply:
Any reason to keep the capsule? Might as well just use a cylinder for the whole thing, no?

Reply:
Cylinder is great for sliding on the ground, not get stuck on things.

Reply:
Ah ok. In this case these are just kinematic colliders for the remote players, so it doesn't matter if they are good at sliding or not, they'll just always follow the remote player around. So I'll just use a cylinder.

Reply:
@amazingalek Never realized but apparently cylinder colliders aren't a thing in Unity by default. I think sticking with capsule is fine for now, no reason to make it too complex.